deploy/helm/kubecf/templates/bosh_deployment.yaml

```diff
@@ -15,14 +15,22 @@ spec:
     name: cf-deployment
     type: configmap
   ops:
-{{- if .Values.features.suse_buildpacks }}
-  - name: {{ include "kubecf.ops-name" "assets/operations/buildpacks/set_suse_buildpacks.yaml" }}
-    type: configmap
-{{- end }}
 {{- if .Values.features.eirini.enabled }}
   - name: ops-use-bits-service
     type: configmap
 {{- end }}
+{{- if eq .Values.features.blobstore.provider "s3" }}
+  - name: ops-use-external-blobstore
+    type: configmap
+  - name: ops-use-s3-blobstore
+    type: configmap
+  - name: ops-configure-bits-service-s3
```

Comment:
I'm getting:

> Error from server: error when creating "dev/kubecf/rendered_template.yaml": admission webhook "validate-boshdeployment.quarks.cloudfoundry.org" denied the request: Failed to resolve manifest: Failed to interpolate ops 'ops-configure-bits-service-s3' for manifest 'kubecf': Applying ops on manifest obj failed in interpolator: Expected to find exactly one matching array item for path '/instance_groups/name=bits' but found 0

Oh, because I'm running _without_ eirini… need to fix the condition above, I guess, to turn on bits service even without eirini in this case.

Reply:
I fixed the condition.